CHANGELOG.md

```diff
@@ -20,19 +20,38 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   Account linking based on emails now require the email to be verified in both users if `shouldRequireVerification` is set to `true` instead of only requiring it for the recipe user.
 -   The access token cookie expiry has been changed from 100 years to 1 year due to some browsers capping the maximum expiry at 400 days. No action is needed on your part.
 -   Recipe functions that update the email address of users now call `isEmailChangeAllowed` to check if the email update should be allowed or not.
+
     -   This only has an effect if account linking is turned on.
     -   This is aimed to help you avoid security issues.
     -   `isEmailChangeAllowed` is now called in functions:
         -   `updateUser` (Passwordless recipe)
         -   `updateEmailOrPassword` (EmailPassword recipe)
         -   `manuallyCreateOrUpdateUser` (ThirdParty recipe)
 
+-   In the multitenancy recipe,
+    -   Removes `emailPasswordEnabled`, `passwordlessEnabled`, `thirdPartyEnabled` inputs from `createOrUpdateTenant` functions.
+    -   Recipe implementation uses v2 APIs for creating, fetching and listing tenants. Refer CDI spec for more information.
+
 ### Changes
 
+-   Adds Multitenancy and Multifactor auth related APIs for dashboard:
+    -   GET `/api/tenants`
+    -   GET `/api/tenant`
+    -   POST `/api/tenant`
+    -   DELETE `/api/tenant`
+    -   PUT `/api/tenant/first-factor`
+    -   PUT `/api/tenant/required-secondary-factor`
+    -   PUT `/api/tenant/core-config`
+    -   GET `/api/thirdparty/config`
+    -   PUT `/api/thirdparty/config`
+    -   DELETE `/api/thirdparty/config`
 -   `passwordResetPOST`:
     -   now verifies the email address in all cases if the EmailVerification recipe is initialized
     -   now tries to link accounts based on account info if AccountLinking is enabled
 -   Extracted some tests into a separate [backend-sdk-testing](https://github.com/supertokens/backend-sdk-testing/) repo, to reuse tests between our backend SDKs
+-   Sends `websiteDomain` and `apiDomain` to core for telemetry.
+-   `boxyURL` is no more mandatory input in `additionalConfig` while adding boxy-saml provider in thirdParty.
+-   Fixes issue with OIDC discover when the input url already contains `.well-known/openid-configuration`.
```

Comment:
other backend sdks will also need these two changes, right?

Why have we changed the boxy hq thing to not need the additional config?